New Bot  drive2022/src/main/java/frc/robot/Robot.java

```diff
@@ -111,42 +110,37 @@
                              // and 1 means that it is just x. 1 is more sensitive and 0 is less sensitive.
   double rightPrime;
   double leftPrime;
-/*
-  //Intake 
-  //values,motors,and motor controller types are subject to change
-  WPI_VictorSPX intakeMotor = new WPI_VictorSPX(5);
-  DoubleSolenoid intakeSol = new DoubleSolenoid(0, PneumaticsModuleType.CTREPCM, 3, 4);
-  Timer intakeTimer = new Timer();
-  // Intake Toggle Variables
-  int intakeToggle = 0;
-  boolean intakeToggleStick;
-  boolean extakeToggleStick;
-  int extakeToggle = 0;
-  // Motor Speed
-  double intakeSpeed = .65;
-
-  //transport
-  WPI_TalonSRX frontBallTransport = new WPI_TalonSRX(10);
-  WPI_TalonSRX backBallTransport = new WPI_TalonSRX(11);
+
+  double tickTock = Timer.getFPGATimestamp();
+
+
+  //ball counter for intake transport/shooter
   DigitalInput ballSwitch = new DigitalInput(3);
-  double ballCount = 0; //this is the initial # of balls in transport at the biginning of a match
+  double ballCount = 1; //this is the initial # of balls in transport at the beginning of a match
```

Comment:
Is there a way to detect the presence of the ball during init, so we don't have to assume that one has been loaded?